archetypes/posts.md

```diff
@@ -1,8 +1,8 @@
 +++
-title = ""
-date = ""
-author = ""
-authorTwitter = "" #do not include @
+title = "{{ replace .Name "-" " " | title }}"
+date = "{{ .Date }}"
+draft = true
+author = '{{ index .Site.Author "name" }}'
```

Comment:
Is it okay to wrap this in a single quote?

Reply:
I did a test with single quotes and it seems to generate just fine. I've changed all string literals to be enclosed by single quotes.